joplin/base/models/site_settings.py

```diff
@@ -41,9 +41,18 @@ class JanisBranchSettings(BaseSetting):
     ]
 
     # Convert branch name to a netlify URL
-    def branch_preview_url_base(self):
+    def url_base(self, janis_branch):
```

Comment:
`janis_branch` is a confusing name for me. It took me a minute to figure out that its a flag to get either the "publish_janis_branch" or the "preview_janis_branch"/"preview_janis_url". It's not the actual name of the janis_branch, which confused me as I read other parts of the code. I like that you've made url_base() work for both previews and publishes, but we need a better name for the janis_branch param.

Reply:
it was just "branch" and was v confusing. Any improvements to code understandability here would be awesome

Reply:
for part of https://github.com/cityofaustin/techstack/issues/3185 I was going to see what it would take to convert these statments to use djangos logging. That way they can be controlled based on env settings and various verbosity stuff

Reply:
"janis_branch_type" ?